src/collections/_documentation/workflow/issue-owners.md

```diff
@@ -3,57 +3,77 @@ title: 'Issue Owners'
 
-In your project settings, you will define which users or teams own which paths or URLs for your application. When an exception is caught that triggers an alert, Sentry evaluates whether the exception’s URL tag matches the URL you specified, or if the path you specified matches any of the paths in the exception’s stack trace.
+### Creating Rules
 
-If there is a match, only Owners will receive the alert for the exception. By default, if there is no match, the alert will be sent to all members of teams associated with this project. You can also choose to specify that in the case of no match, no users be notified, by toggling off the switch in project settings > Issue Owners > ‘If ownership cannot be determined for an issue...’
+You define ownership rules per project. To configure ownership rules, navigate to your **Project Settings > Issue Owners,** or click on the "Create Ownership Rule" button on an issue details page.
 
-[{% asset owners_default_everyone.png %}]({% asset owners_default_everyone.png @path %})
+There are three types of matchers available:
 
-At this time, the Issue Owners feature is only available for email notifications. This means that your alert rules must trigger email notifications to be affected by your Issue Owners' rules.
+1. Path: matches against all file paths in the event's stack trace
+2. URL: matches against the event's `url` tag
+3. Tag: matches against event tags
 
-## Configuration
+The general format of a rule is: `type:pattern owners`
 
-**Adding a New Rule**
+`type`
 
-To configure Issue Owners, navigate to your project settings > Issue Owners.
+can be either `path`, `url`, or `tags.TAG_NAME`, depending on whether you want to match on paths, URL, or a specific tag.
 
-To add a new rule, you can use the dropdown to specify whether you’re using a path or URL. Then add the path or URL, and specify which users or teams own that path. You can add multiple users or teams to one path.
+`pattern`
 
-You can also use the text editor below to manually add rules.
+The pattern you're matching on (for example, `src/javascript/*` for `path`, `[https://www.example.io/checkout](https://www.example.io/checkout)` for `url`, or `Chrome 81.0.*` for `tags.browser`. It supports unix-style [glob syntax](https://en.wikipedia.org/wiki/Glob_(programming)). For example, `*` to match anything and `?` to match a single character. *This is not a regex.*
 
-[{% asset owners_panel.png %}]({% asset owners_panel.png @path %})
+`owners` 
 
-You can also add a new rule from an individual issue. From the issue details, click the ‘Create Ownership Rule’ on the right-hand panel.
+The single owner or space-separated list of owners. Each owner is the email of a Sentry user, or the name of a team prefixed with `#` (for example, `#backend-team`).
```

Comment:
Suggest to clarify: "The owner, or list of owners (created by listing each with a space separating each owner) identified by either the email of the Sentry user or the name of the team, prefixed with `#`. Thus, the owner may be example@company.com or example1@company.com example2@company.com or #backend-team@company.com."